Automattic/vip-go-mu-plugins
# Adds safe reindexing support

## Description
This change makes it easier to reindex life sites without impacting returned results.

## Changelog Description

### Plugin Updated: Search

Adds a `--using-versions` to index subcommand. This enables caller to call for instance:

```
wp vip-search index --using-versions --indexables=post --setup
```

This command will create a new version of index and fully index the content. While this is happening the previous version of index is still active and therefore not impacted and serving correct content. After indexing finishes, the command will activate the newly indexed version and remove the previous version. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. run `wp vip-search index --using-versions --indexables=post --setup`
1. run `wp vip-search index-versions list post`and see the new version replaced the previous one.
